src/app/pages/storage/modules/devices/stores/devices-store.service.ts

```diff
@@ -114,6 +117,18 @@ export class DevicesStore extends ComponentStore<DevicesState> {
     );
   });
 
+  readonly loadDisksWithSmartSupport = this.effect((triggers$: Observable<void>) => {
+    return triggers$.pipe(
+      switchMap(() => {
+        return this.ws.call('smart.test.disk_choices').pipe(
+          tap((disksWithSmartSupport) => {
+            this.patchState({ disksWithSmartSupport: Object.values(disksWithSmartSupport) });
```

Comment:
Would be good to keep naming more specific. Here for example, it's not just disks with smart support, it's disks that support running a smart test.